### DISCUSSION TITLE
Show actual reviews, but use a different text column in the corpus creation

### DISCUSSION THREAD

**Discussion by AnubhavCR7** [JasonKessler/scattertext#96] [Q&A]
Hello,
I am using the ScatterText plot the keywords extracted from different reviews. But instead of using the actual review text, I am using another column in the dataframe, called 'key_text' which is basically a string made by concatenating the top 3 keywords extracted from a review. Like if the keywords extracted for a review are 'wait', 'come' and 'go', then the corresponding string will be 'wait come go'. The keywords are extracted using a different algorithm, i.e., I am not using the inbuilt Spacy model for keyword extraction. 

But when I search for a term in the ScatterText plot, the texts present in 'key_text' column is displayed, corresponding to that search term in the corpus. 

1. Is there a way where I can show the actual reviews present in the data when someone searches for a specific term, but use the 'key_text' column in this line of code :

corpus = scattertext.CorpusFromPandas(data, category_col = 'Category', text_col = 'key_text', nlp = nlp).build()
              .remove_terms(nlp.Defaults.stop_words, ignore_absences = True) 

2. Is there a way we can disable the search option in the ScatterText plot ?

Regards.

**Comment by JasonKessler** [comment]
1. You can experiment with the alternative_text_field argument to produce_scattertext_explorer. Put it in your original dataframe, and use CorpusFromParsedDocuments to make your corpus. Not sure if this still works. Feel free to modify the code to get it work if it doesn't. 
2. Not at this time.
